Makefile

```diff
@@ -60,3 +60,13 @@ test-suite test-tox:
 	PATH="${PWD}/venv/bin:${PATH}" tox -e $(TOX_ENVLIST) -- $(TOX_ARGS)
 
 test-all: test-unit test-integration
+
+pip-compile: venv/bin/pip-compile
+	venv/bin/pip-compile --allow-unsafe --generate-hashes --output-file=requirements.txt requirements.in
+	venv/bin/pip-compile --generate-hashes --output-file=requirements-web.txt requirements-web.in
+	venv/bin/pip-compile --allow-unsafe --generate-hashes --output-file=requirements-test.txt requirements-test.in
+
+venv/bin/pip-compile:
+	# pip-compile>=6 fails with
+	#   "option --use-deprecated: invalid choice: 'legacy-resolver' (choose from )"
```

Comment:
Do we have a follow up issue for addressing this?

Reply:
Not yet, in fact I can't seem to reproduce it anymore. I'll try to figure out what changed.

Reply:
`pip < 20.3` does not support `--use-deprecated legacy-resolver`
`pip >= 20.3` does

pip-tools does specify this dependency properly: https://github.com/jazzband/pip-tools/blob/f49b3fa8aa1b5d8c6dda80f719eaae706feff6b0/setup.cfg#L34

I must have messed something up while testing. I removed the `<6` constraint.